components/brave_wallet/browser/solana_transaction.h

```diff
@@ -69,6 +75,7 @@ class SolanaTransaction {
  private:
   FRIEND_TEST_ALL_PREFIXES(SolanaTransactionUnitTest, GetBase64EncodedMessage);
   SolanaMessage message_;
+  std::vector<uint8_t> signatures_;
```

Comment:
Please put a comment to say this is only assigned when `FromSignedTransactionBytes` is called.

Reply:
fixed and squashed in 88c253f4b44380b67a6c8c3201f51e04d31e9d4e